server/text_generation_server/models/__init__.py

```diff
@@ -110,6 +111,138 @@
     __all__.append(Mamba)
 
 
+class ModelType(enum.Enum):
+    MAMBA = {
+        "type": "ssm",
+        "name": "Mamba",
+        "url": "https://huggingface.co/state-spaces/mamba-2.8b-slimpj",
```

Comment:
Maybe `example_repo` or something like that?

Reply:
It's just to output a link in the doc. Could be the paper page or anything. In here I'd like to keep things minimal.